android/src/main/java/com/amplitude/android/Timeline.kt

```diff
@@ -61,52 +71,42 @@ class Timeline(
         var skipEvent = false
 
         if (event.eventType == Amplitude.START_SESSION_EVENT) {
-            setSessionId(eventSessionId ?: eventTimestamp)
-            refreshSessionTime(eventTimestamp)
+            session.setSessionId(eventSessionId ?: eventTimestamp)
```

Comment:
Question: Does this need? 
When we fire the `START_SESSION_EVENT` we already fire the `setSessionId` which is in android/src/main/java/com/amplitude/android/utilities/Session.kt in line #97?